Makefile

```
@@ -290,24 +290,12 @@ vulncheck:
 
 .PHONY: tidy-all
 tidy-all:
-	cd api && $(GO_CMD) mod tidy
-	cd api/auth/approle && $(GO_CMD) mod tidy
-	cd api/auth/kubernetes && $(GO_CMD) mod tidy
-	cd api/auth/ldap && $(GO_CMD) mod tidy
-	cd api/auth/userpass && $(GO_CMD) mod tidy
-	cd sdk && $(GO_CMD) mod tidy
-	$(GO_CMD) mod tidy
+	bash -c 'find . -name go.mod | while read go_mod; do ( cd "$$(dirname "$$go_mod")" && go mod tidy ) ; done'
```

I think this is POSIX enough (same for ci-tidy-all):

```suggestion
	sh -c 'find . -name go.mod | while read go_mod; do ( cd "$$(dirname "$$go_mod")" && go mod tidy ) ; done'
```